app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -0,0 +1,262 @@
+import React from 'react'
+import { isEmpty } from 'lodash'
+import { Form } from 'react-form'
+
+import TableWrapper from '~/components/atoms/TableWrapper'
+import ExpandableTable from '~/components/molecules/ExpandableTable'
+import Button from '~/components/atoms/Button'
+import { withContext } from '../context'
+import FormGrid from '~/components/molecules/FormGrid'
+import ExpandablePanel from '~/components/molecules/ExpandablePanel'
+import YesNoRadioGroup from '../YesNoRadioGroup'
+import formUtils from '~/utils/formUtils'
+import { withField, Field } from '~/utils/form/Field'
+
+const { ExpanderButton } = ExpandableTable
+
+const typeOfAssistance = [
+  'Catholic Charities',
```

Comment:
Yes, for now I think it makes sense to have them here, and it would be good to follow up with SMEs about the frequency with which this changes, so we can plan for if/when to store/draw these from Salesforce.